.travis.yml

```diff
@@ -1,8 +1,11 @@
-dist: trusty
+dist: xenial
 language: java
-jdk: oraclejdk8
+jdk: openjdk8
 
 addons:
+  apt:
+    packages:
+    - nginx
```

Comment:
`nginx` is needed for our test HTTP/2 kitchen service. The test service run standard HTTP/1 kitchen fronted by an HTTP/2 compliant nginx proxy.